services/QuillLMS/Gemfile

```diff
@@ -5,7 +5,7 @@ source 'https://rubygems.org'
 ruby '2.7.6'
 
 # CORE DEPS
-gem 'rails', '5.2.8'
+gem 'rails', '~> 6.0.5'
```

Comment:
I've viewed patch upgrades even with Rails as safe updates, but I will change it back to the locked version since I don't really have a strong preference.